docs/versions.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Version Policy
+---
+
+# Version Policy
+
+ParchmentMC maintains a version policy to specify what versions of Minecraft are prioritized by the ParchmentMC team 
+for actively reviewing contributions. 
+
+Due to the technical and design requirements of the toolchain, Parchment only supports versions of Minecraft which has 
+published obfuscation logs.
+
+Each major version or release of Minecraft which Parchment maintains a mapping set for will have one branch. All minor 
+versions associated with that major versions are included into that one branch. Only the latest minor version of each 
+major release will be supported.
+
+The latest minor version of the latest major release will be the actively reviewed version, and is the recommended 
```

Comment:
re: discord discussion, this needs to be clearer about which *major* version or versions we support and how we decide it :P

Reply:
As clarified by @Lanse505, we support the two most latest major releases. I've also removed the phrase 'latest minor version', to both not distract the reader from the real meaning of the sentence ("what major versions are actively reviewed"), and because the last sentence of the previous paragraph already states that only the latest minor version of the latest major release is supported.